frontend/src/pages/patients/PatientsPage.tsx

```diff
@@ -31,17 +31,17 @@ export type PatientIdsTriplet = {
 };
 
 function patientAliasFilterWhereVariables(
-  uniqueQueries: string[]
+  parsedSearchVals: string[]
```

Comment:
Renamed this function argument to improve clarity as this function only takes in the state value named `parsedSearchVals`.